docs/developer_documentation/app-development.md

```diff
@@ -0,0 +1,130 @@
+# MOLGENIS App store
+
+You can make your own app in MOLGENIS using the appstore.
+
+You can perform the following steps to get your app in the App store.
+
+- [Create a project](#create-a-project)
+- [Update your build configuration](#update-build-configuration)
+- [Update your language specific configuration](#update-your-language-specific-configuration)
+- [Create a freemarker-template](#create-a-freemarker-template)
+- [Upload your app](#upload-your-app)
+- [Active the app](#active-the-app)
+
+## Create a project
+You can create a javascript project in a language you like. 
+We develop our own apps in VUE so most documentation will point out the VUE-configuration.
+
+## Update your build configuration
+You must update build configuration to build a zip-file which you can upload in MOLGENIS. 
+At the moment we only describe how to do this in webpack.
+
+### webpack
+In your config/index.js
+
+```
+ assetsRoot: path.resolve(__dirname, '../dist/static'),
+ assetsSubDirectory: '',
+ assetsPublicPath: '/apps/${app.id}',
+ 
+ /**
+  * Source Maps
+  */
+ 
+ productionSourceMap: true,
+ // https://webpack.js.org/configuration/devtool/#production
+ devtool: '#eval-source-map',
+```
+
+## Update your language specific configuration
+You have to update some  configuration in your app to make it work in MOLGENIS. 
+We now only describe how to do this in VUE.
+
+### VUE
+
+- Add the following code-snippet in src/main.js
+
+```
+if (window.__webpack_public_path__) {
+  /* eslint-disable no-undef, camelcase */
+  __webpack_public_path__ = window.__webpack_public_path__
+  /* eslint-enable */
+}
+```
+
+- Add the following code-snippet in src/router/index.js
+
+```
+...
+const {baseUrl} = window.__INITIAL_STATE__ || {}
+
+export default new Router({
+  mode: 'history',
+  base: baseUrl,
```

Comment:
You can only mention that base parameter should be set, routes and mode is not relevant